esqueleto.cabal

```
@@ -2,7 +2,7 @@ cabal-version: 1.12
 
 name:           esqueleto
 
-version:        3.5.5.0
+version:        3.6.0.0
```

Since all changes here are an addition of a new module, then we can make this a minor change instead of a major change.

I don't *think* we've landed anything for 3.6 yet, so this can even get released now 😄 
```suggestion
version:        3.5.6.0
```